landavailability/api/management/commands/utils.py

```diff
@@ -0,0 +1,12 @@
+from api.models import Address, CodePoint
+
+
+def get_address_from_postcode(postcode):
+    return Address.objects.filter(postcode=postcode)
```

Comment:
Data in Address and Codepoint is normalised at import time, but input could be different. Good spot. Going to update it